### ISSUE TITLE
Validation Error: Image Width

### ISSUE THREAD

**Issue by cncoulter** [cncoulter/cncoulter.github.io#2]
"Error: Bad value 40% for attribute width on element img: Expected a digit but saw % instead."

The photo on the about page should have its width in px, not percent.

**Comment by cncoulter** [comment]
Fixed!

https://github.com/cncoulter/cncoulter.github.io/commit/a7056a88a1988dfc66595ea31bb934c58f00cdeb
